src/main/java/uk/gov/companieshouse/api/accounts/service/impl/TnepValidationServiceImpl.java

```diff
@@ -55,19 +55,10 @@ public boolean validate(String ixbrl, String location) {
             Results results = validatIxbrlAgainstTnep(ixbrl, location);
 
             if (hasPassedTnepValidation(results)) {
-                addToLog(false, null, results, location,
```

Comment:
We should still log this, we only need omit the entire results object from the log. That's what's causing the logging issues

Reply:
I rolled back and removed only "results" from the **addToLog**